GitUI/Strings.cs

```diff
@@ -13,6 +13,7 @@ internal sealed class Strings : Translate
         private readonly TranslationString _okText = new TranslationString("OK");
         private readonly TranslationString _cancelText = new TranslationString("Cancel");
 
+        private readonly TranslationString _containedInCurrentCommitText = new TranslationString("{0} is contained in the currently selected commit");
```

Comment:
I don't know if it's not better to use `"'{0}' is contained in the currently selected commit"` (single quoted name of branch)?